### PR TITLE
Add "How it works" page

### PR DESCRIPTION
**Pull request by shpandrak**
Create new how it works page
Remove architecture page
Fix all pages references to point at the "how it works" page

### CODE REVIEW

File: how-it-works.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+                <br/>
+                <br/>
+                In order to make a copy of the entire hackathon application state and later restore it on a different
+                environment,
+                Ocopea needs the ability to make copies of each individual component configuration and data:
+                <br/>
+            </p>
+            <ul class="list">
+                <li>
+                    The hackathon docker image version that is used including the configuration (scale, environment
```
Comment (mrmccarthy) [inline how-it-works.html]: If I deploy an app via Ocopea and then scale up one of the microservices before making an app copy, will the same scale be applied to a restored copy?

Comment (shpandrak) [reply how-it-works.html]: No, but it is a bug :)

Comment (shpandrak) [reply how-it-works.html]: I can omit "scale" if you prefer, but I prefer opening a bug...

Comment (mrmccarthy) [reply how-it-works.html]: If we have code that should scale but got broken somehow, fine - a bug.
If we never had it working, then we should remove scale.

File: how-it-works.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+                />
+            </div>
+
+            <br/>
+            <br/>
+            <div class="title">PSB</div>
+            <p>
+                The PSB is responsible for communicating with the PaaS or the container orchestrator. It is used by
+                the Ocopea site to deploy the stateless containers and bind them to infrastructure services
+                (Data Services)
```
Comment (vjtirumalai) [inline how-it-works.html]: Is Data Service actually a DB say in hackathon case?

Comment (shpandrak) [reply how-it-works.html]: DB is a kind of data service...

Comment (vjtirumalai) [reply how-it-works.html]: Ok thanks. If Mike is okay with your changes will push in then.

File: how-it-works.html (hunk 1 above)
Comment (shpandrak) [reply how-it-works.html]: @mrmccarthy I have made the changes and squashed all commits (thanks for the typos fix 👍 ) please hit "approve" so I can merge

**Pull request merged**